gio-subclass/src/application.rs

```diff
@@ -14,13 +14,13 @@ use gobject_subclass::anyimpl::*;
 use gobject_subclass::object::*;
 
 pub struct ArgumentList{
```

Comment:
Probably want to implement `Into<Vec<String>>` for this, and definitely `Debug` (directly call the `Debug` impl of the `Vec`)